src/styles/main.scss

```diff
@@ -1,3 +1,206 @@
+  box-sizing: border-box;
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+
+  &__card {
+    margin-right: 48px;
+    margin-bottom: 46px;
+  }
+
+  &__card:nth-child(4n) {
+    margin-right: 0;
+  }
+
+  &__card:nth-last-child(-n+4) {
+    margin-bottom: 0;
+  }
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+  display: block;
+}
+
+.nav {
+  font-weight: 500;
+
+  &__list {
+    list-style: none;
+    @include displayFlexR();
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item:not(:first-child) {
+    margin-left: 20px;
+  }
+
+  &__link {
+    color: $text-color;
+    font-family: inherit;
+    font-size: 12px;
+    font-weight: 500px;
+    text-decoration: none;
+    height: 60px;
+    @include displayFlexRAlignItemsC();
+    text-transform: uppercase;
+    position: relative;
+  }
+
+  &__link--active {
+    color: $button-color;
+  }
+
+  &__link--active::after {
+    position: absolute;
+    content: "";
+    bottom: 0;
+    left: 0;
+    right: 0;
+
+    height: 4px;
+    border-radius: 2px;
+    background-color: $button-color;
+  }
+
+  &__link:hover {
+    color: #00acdc;
+  }
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  background: #fff;
+
+  &__image {
+    display: block;
+    height: 134px;
+    margin: 0 auto 40px;
+    width: 160px;
+  }
+
+  &__title {
+    color: $text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  }
+
+  &__review-container {
+    @include displayFlexR();
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review-count {
+    color: $text-color;
+  }
+
+  &__price-container {
+    @include displayFlexRAlignItemsC();
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__price {
+    color: $text-color;
+    font-size: 16px;
+    font-weight: bold;
+    line-height: 18px;
+  }
+
+  &__buy {
+    @include displayFlexRAlignItemsC();
+    justify-content: center;
+    font-size: 14px;
+    line-height: 16px;
+    text-transform: uppercase;
+    color: #fff;
+    background-color: $button-color;
+    font-weight: 700;
+    box-sizing: border-box;
+    width: 166px;
+    height: 40px;
+    border: 1px solid $button-color;
+    border-radius: 5px;
+    cursor: pointer;
+    padding: 0;
+  }
+
+  &__buy:hover {
+    color: $button-color;
+    background-color: #fff;
+  }
+}
+
+.stars {
+  @include displayFlexR();
+
+  &__star {
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+
+  &--4 .stars__star:last-child {
+    background-image: url(../images/star.svg);
+  }
```

Comment:
make styles for each amount of active stars
and please pay attention on checklist point
![Screenshot from 2023-04-15 15-35-03](https://user-images.githubusercontent.com/33029864/232224008-63b4c5f1-bb88-4ccc-9261-9ced5dfafac9.png)

Reply:
Thank you,

I just changed to @for

Reply:
it should be to 5. it real site should be possibilty to set also 5 active stars.